lib/logthrdestdrv.c

```diff
@@ -130,19 +139,47 @@ __disconnect(LogThrDestDriver *self)
   self->worker.connected = FALSE;
 }
 
-
-
+/* NOTE: runs in the worker thread */
 static void
-_disconnect_and_suspend(LogThrDestDriver *self)
+_disconnect_and_suspend(LogThreadedDestDriver *self)
 {
   self->suspended = TRUE;
-  __disconnect(self);
+  _disconnect(self);
   log_queue_reset_parallel_push(self->queue);
-  log_threaded_dest_driver_suspend(self);
+  _suspend(self);
 }
 
+/* NOTE: runs in the worker thread */
+void
```

Comment:
Make these function `static`.
It can be done in your other PR, so we can merge this as-is.